ci/configure-deployment-parameters.sh

```diff
@@ -71,6 +75,8 @@ function check-frontend-params {
   check-parameter-set "${parameter}" || write-parameter-value "$parameter" "$([[ $ACCOUNT == production ]] && echo false || echo true)"
   parameter=${PARAMETERS[admin_tool_url]}
   check-parameter-set "${parameter}" || write-parameter-value "$parameter" "https://admin.sign-in.service.gov.uk"
+  parameter=${PARAMETERS[use_stub_servicenow]}
+  check-parameter-set "${parameter}" || write-parameter-value "$parameter" "true"
```

Comment:
check-parameter-set "${parameter}" || write-parameter-value "$parameter" "false"

Reply:
Not sure why you have requested this one to be set to False when other such Parameters are set to True but changed as requested.